iam/validate.go

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2020 The Things Industries B.V.
+
+package iampb
+
+import (
+	"errors"
+
+	packetbroker "go.packetbroker.org/api/v3"
+)
+
+// Validate returns whether the request is valid.
+func (r *CreateNetworkRequest) Validate() error {
+	if r.Network == nil {
```

Comment:
I think we should start following the rule of either doing a `nil`-check for `r` here, or using `GetNetwork()` etc.

Same below and in the other `validate.go`'s